CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ set(POSITION_INDEPENDENT_CODE True)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_CXX_FLAGS
-    "${CMAKE_CXX_FLAGS} -mavx2 -mfma -mavx -mf16c -mbmi2 -march=native -mno-avx512f -D USE_VELOX_COMMON_BASE"
+    "${CMAKE_CXX_FLAGS} -mavx2 -mfma -mavx -mf16c -mbmi2 -mbmi -march=x86-64 -mno-avx512f -D USE_VELOX_COMMON_BASE"
```

Comment:
If we are not using `native`, we can remove the `no-avx512f` as well.
Also `x86-64` is default on Intel and AMD so you can omit that.